mysite/fsm/base_fsm_node.py

```diff
@@ -0,0 +1,72 @@
+from django.core.urlresolvers import reverse
+from django.utils.safestring import mark_safe
+from chat.models import Message, EVAL_OPTIONS, STATUS_OPTIONS
+from ct.models import UnitStatus, UnitLesson, Lesson, Response
+# from chat.models import Chat, Message, ChatDivider, UnitError
+import logging
+import re
+from functools import partial
+from ct.templatetags.ct_extras import md2html
+
+
+class BaseFMSNode(object):
+    """
+    Base class for all FSM nodes which implements logic to get html (get_html method), get options (get_options)
```

Comment:
Docstring formatting - do no exceed 80 chars please.

Reply:
First line for the title. Then blank line. Then additional info.

Reply:
why not 120 chars? we all have wide displays and 80 symbols is very short string... I thought we are using 120 chars/line